infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -284,7 +286,7 @@ exports.tasks.push({
             MINIO_ROOT_PASSWORD: 'miniopassword',
           },
         }),
-        ui: serviceDefinition('ui', { command: ['ui/web'] }),
+        ui: serviceDefinition('ui'),
```

Comment:
Do you remember why this command was removed? Now when you simply run docker compose up it will fail to start ui container because it lacks entrypoint command